### DISCUSSION TITLE
renaming Definitely Typed default branch to main

### DISCUSSION THREAD

**Discussion by johnnyreilly** [DefinitelyTyped/DefinitelyTyped#53605] [_Misc]
Heya! 

Apologies if this has already been covered in issues, I did a little digging but didn't spot anything.

Is there a desire to rename the default branch of Definitely Typed default branch to `main` from `master`? I won't go into the reasons why, [others have covered that better](https://www.hanselman.com/blog/easily-rename-your-git-default-branch-from-master-to-main) already.

My guess is that this might break a number of the integrations built on top of DT.  And so it probably wouldn't be entered into lightly. But I guess the first question is this: is this something folk would like to do?

**Comment by sandersn** [comment]
Sounds good to me. github.com/microsoft/DefinitelyTyped-tools contains the bulk of what the TS team would need to change. dtslint and some of the watchdogs might also need to change.
